ui/app/AppLayouts/Communities/views/MintedTokensView.qml

```diff
@@ -254,7 +254,9 @@ StatusScrollView {
                     navigationIconVisible: false
                     privilegesLevel: model.privilegesLevel
                     ornamentColor: model.color
-                    communityId: root.communityId
+
+                    // https://www.figma.com/file/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?type=design&node-id=29384-566477&mode=design&t=SND7hSzPaI2pswM1-0
```

Comment:
Maybe instead of adding the figma file, you can add just a comment, since figma can dynamically change more than the spec itself! :)